

<launch>
    <node name="pressure_visualizer" pkg="m3skin_viz"
        type="m3skin_viz" output='screen' >
        <remap from='skin_patch_forearm_right' to='fabric_skin' />
        <remap from='/skin_patch_forearm_right/taxels/srv/local_coord_frames'
            to='/fabric_skin/taxels/srv/local_coord_frames'
            />
        <remap from='/skin_patch_forearm_right/taxels/srv/link_name'
            to='/fabric_skin/taxels/srv/link_name' />
        <remap from='/skin_patch_forearm_right/taxels/forces'
            to='/fabric_skin/taxels/forces'/>
    </node>
</launch>

